### PR TITLE
fix PEP8 issues in assign_wcs, enable an IFU test again

### PR DESCRIPTION
**Pull request by nden**
#2340 

This actually fixed some real problems discovered by flake8!

### CODE REVIEW

File: jwst/assign_wcs/nircam.py [hunk 1]
```diff
@@ -139,7 +139,6 @@ def tsgrism(input_model, reference_files):
         dispx = f.dispx
         dispy = f.dispy
         invdispx = f.invdispx
-        invdispy = f.invdispy
```
Comment (nden) [inline jwst/assign_wcs/nircam.py]: @sosey Is it OK to remove this?

Comment (sosey) [reply jwst/assign_wcs/nircam.py]: no. it is necessary.

Comment (sosey) [reply jwst/assign_wcs/nircam.py]: It's used for the column grism model in WFSS

Comment (nden) [reply jwst/assign_wcs/nircam.py]: Oh, I see. But that  is a different function. This one here is `tsgrism` so perhaps can be removed?

Comment (sosey) [reply jwst/assign_wcs/nircam.py]: ah - when I clicked it was taking me to the new file without the line, and only the wfss part...I had missed the grayed out text suggesting it was in tsgrism in the  master...yah, you can remove it. I've already removed it my PR for this mode as well.

**Pull request merged**